services/QuillLMS/app/workers/snapshots/cache_snapshot_count_worker.rb

```diff
@@ -35,13 +35,16 @@ def perform(cache_key, query, user_id, timeframe, school_ids, filters)
 
       Rails.cache.write(cache_key, payload, expires_in: cache_expiry)
 
-      SendPusherMessageWorker.perform_async(user_id, PUSHER_EVENT,
-        {
-          query: query,
-          timeframe: timeframe['name'],
-          school_ids: school_ids
-        }.merge(filters)
-      )
+      filter_hash = Digest::MD5.hexdigest([
```

Comment:
Let's make this a separate service and test it explicitly with generic data, e.g.
```ruby
digest = HashDigest.run(array)